<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QWidget" name="tabCustomer">
       <attribute name="title">
        <string> 客户管理</string>
       </attribute>
       <widget class="QLabel" name="cname_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> 客户名称：</string>
        </property>
       </widget>
       <widget class="QLabel" name="ctype_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> 客户类型：</string>
        </property>
       </widget>
       <widget class="QLabel" name="cthone_label">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>30</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>电话：</string>
        </property>
       </widget>
       <widget class="QLabel" name="address_label">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>60</y>
          <width>50</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string> 地址：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cname">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>30</y>
          <width>200</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cpthone">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>30</y>
          <width>300</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_caddress">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>60</y>
          <width>300</width>
          <height>60</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_ctype">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>70</y>
          <width>200</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableViewCus">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>751</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>761</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string> 客户管理</string>
        </property>
        <widget class="QPushButton" name="showCusBtn">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>140</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string> 显示全部</string>
         </property>
        </widget>
        <widget class="QPushButton" name="inportCusBtn">
         <property name="geometry">
          <rect>
           <x>268</x>
           <y>140</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string> 导入客户</string>
         </property>
        </widget>
        <widget class="QPushButton" name="searchCusBtn">
         <property name="geometry">
          <rect>
           <x>349</x>
           <y>140</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string> 搜索</string>
         </property>
        </widget>
        <widget class="QPushButton" name="createcusBtn">
         <property name="geometry">
          <rect>
           <x>177</x>
           <y>139</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string> 新建客户</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>140</y>
           <width>239</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>209</y>
          <width>851</width>
          <height>330</height>
         </rect>
        </property>
        <property name="title">
         <string> 客户显示---（温馨提示：双击单元格即可编辑）</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delCusBtn">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>删除选中行</string>
        </property>
       </widget>
       <zorder>groupBox_2</zorder>
       <zorder>groupBox</zorder>
       <zorder>cname_label</zorder>
       <zorder>ctype_label</zorder>
       <zorder>cthone_label</zorder>
       <zorder>address_label</zorder>
       <zorder>lineEdit_cname</zorder>
       <zorder>lineEdit_cpthone</zorder>
       <zorder>textEdit_caddress</zorder>
       <zorder>comboBox_ctype</zorder>
       <zorder>tableViewCus</zorder>
       <zorder>delCusBtn</zorder>
      </widget>
      <widget class="QWidget" name="tabSupplier">
       <attribute name="title">
        <string> 供应商管理</string>
       </attribute>
       <widget class="QComboBox" name="comboBox_stype">
        <property name="geometry">
         <rect>
          <x>75</x>
          <y>70</y>
          <width>200</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_saddress">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>60</y>
          <width>300</width>
          <height>60</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>209</y>
          <width>871</width>
          <height>330</height>
         </rect>
        </property>
        <property name="title">
         <string>  供应商显示---（温馨提示：双击单元格即可编辑）</string>
        </property>
        <widget class="QTableView" name="tableViewSup">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>751</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="delSupBtn">
         <property name="geometry">
          <rect>
           <x>780</x>
           <y>100</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>删除选中行</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="stype_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> 客户类型：</string>
        </property>
       </widget>
       <widget class="QLabel" name="saddress_label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>50</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string> 地址：</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>9</y>
          <width>761</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string> 供应商管理</string>
        </property>
        <widget class="QPushButton" name="showSupBtn">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>140</y>
           <width>77</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string> 显示全部</string>
         </property>
        </widget>
        <widget class="QPushButton" name="inportSupBtn">
         <property name="geometry">
          <rect>
           <x>255</x>
           <y>140</y>
           <width>77</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string> 导入供应商</string>
         </property>
        </widget>
        <widget class="QPushButton" name="searchSupBtn">
         <property name="geometry">
          <rect>
           <x>338</x>
           <y>140</y>
           <width>76</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string> 搜索</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_sname">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="createsupBtn">
         <property name="geometry">
          <rect>
           <x>174</x>
           <y>139</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string> 新建供应商</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>140</y>
           <width>244</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </widget>
       </widget>
       <widget class="QLabel" name="sname_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> 供应商名称：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_sphone">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>30</y>
          <width>300</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="sphone_label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>电话：</string>
        </property>
       </widget>
       <zorder>groupBox_4</zorder>
       <zorder>comboBox_stype</zorder>
       <zorder>textEdit_saddress</zorder>
       <zorder>groupBox_3</zorder>
       <zorder>stype_label</zorder>
       <zorder>saddress_label</zorder>
       <zorder>sname_label</zorder>
       <zorder>lineEdit_sphone</zorder>
       <zorder>sphone_label</zorder>
      </widget>
      <widget class="QWidget" name="tabGood">
       <attribute name="title">
        <string> 货品管理</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="pronum_label">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>31</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> 货品编码：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pronum_lineEdit">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>31</y>
          <width>130</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="proname_lineEdit">
        <property name="geometry">
         <rect>
          <x>311</x>
          <y>31</y>
          <width>130</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="proname_label">
        <property name="geometry">
         <rect>
          <x>239</x>
          <y>31</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> 货品名称：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prospec_lineEdit">
        <property name="geometry">
         <rect>
          <x>519</x>
          <y>31</y>
          <width>130</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="prospec_label">
        <property name="geometry">
         <rect>
          <x>447</x>
          <y>31</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> 货品规格：</string>
        </property>
       </widget>
       <widget class="QLabel" name="probarcode_label">
        <property name="geometry">
         <rect>
          <x>655</x>
          <y>31</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> 条  码：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="probarcode_lineEdit">
        <property name="geometry">
         <rect>
          <x>715</x>
          <y>31</y>
          <width>130</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="probatno_label">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>80</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>批     号：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="probatno_lineEdit">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>80</y>
          <width>130</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="prounit_label">
        <property name="geometry">
         <rect>
          <x>239</x>
          <y>80</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>单    位：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prouint_lineEdit">
        <property name="geometry">
         <rect>
          <x>311</x>
          <y>80</y>
          <width>130</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="procount_label">
        <property name="geometry">
         <rect>
          <x>447</x>
          <y>80</y>
          <width>66</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>数     量：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="procount_spinBox">
        <property name="geometry">
         <rect>
          <x>519</x>
          <y>80</y>
          <width>130</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
       <widget class="QLabel" name="proprice_label">
        <property name="geometry">
         <rect>
          <x>655</x>
          <y>80</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>单  价：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="proprice_spinBox">
        <property name="geometry">
         <rect>
          <x>715</x>
          <y>80</y>
          <width>130</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
       <widget class="QLabel" name="prototalprice_label">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>120</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>金     额：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="prostack_spinBox">
        <property name="geometry">
         <rect>
          <x>311</x>
          <y>120</y>
          <width>130</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="prototalprice_spinBox">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>120</y>
          <width>130</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
       <widget class="QLabel" name="prostack_label">
        <property name="geometry">
         <rect>
          <x>239</x>
          <y>120</y>
          <width>66</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>库存数量：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="prolack_spinBox">
        <property name="geometry">
         <rect>
          <x>519</x>
          <y>120</y>
          <width>130</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QLabel" name="prolack_label">
        <property name="geometry">
         <rect>
          <x>447</x>
          <y>120</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>缺少数量：</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>871</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>货品管理</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>170</y>
           <width>336</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="createpro_Btn">
            <property name="text">
             <string> 新建货品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="inportProBtn">
            <property name="text">
             <string> 导入货品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchProBtn">
            <property name="text">
             <string> 搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="showProBtn">
            <property name="text">
             <string> 显示全部</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>871</width>
          <height>330</height>
         </rect>
        </property>
        <property name="title">
         <string>货品显示---（温馨提示：双击单元格即可编辑）</string>
        </property>
        <widget class="QTableView" name="tableViewPro">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>751</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="delProBtn">
         <property name="geometry">
          <rect>
           <x>780</x>
           <y>100</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>删除选中行</string>
         </property>
        </widget>
       </widget>
       <zorder>groupBox_5</zorder>
       <zorder>label</zorder>
       <zorder>pronum_label</zorder>
       <zorder>pronum_lineEdit</zorder>
       <zorder>proname_lineEdit</zorder>
       <zorder>proname_label</zorder>
       <zorder>prospec_lineEdit</zorder>
       <zorder>prospec_label</zorder>
       <zorder>probarcode_label</zorder>
       <zorder>probarcode_lineEdit</zorder>
       <zorder>probatno_label</zorder>
       <zorder>probatno_lineEdit</zorder>
       <zorder>prounit_label</zorder>
       <zorder>prouint_lineEdit</zorder>
       <zorder>procount_label</zorder>
       <zorder>procount_spinBox</zorder>
       <zorder>proprice_label</zorder>
       <zorder>proprice_spinBox</zorder>
       <zorder>prototalprice_label</zorder>
       <zorder>prostack_spinBox</zorder>
       <zorder>prototalprice_spinBox</zorder>
       <zorder>prostack_label</zorder>
       <zorder>prolack_spinBox</zorder>
       <zorder>prolack_label</zorder>
       <zorder>groupBox_6</zorder>
      </widget>
      <widget class="QWidget" name="tabSell">
       <attribute name="title">
        <string> 进货管理</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabPurchase">
       <attribute name="title">
        <string>销货 管理</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
